app/controllers/reports_controller.rb

```diff
@@ -101,6 +111,18 @@ def bulk_recommend_export_filename
     "#{time_now}_bulk-recommend_register-trainee-teachers.csv"
   end
 
+  def bulk_placement_trainees
+    current_user.organisation.trainees_to_be_placed
+  end
+
+  def bulk_placement_export_filename
+    "#{time_now}_#{organisation_name_snake_case}-to-add-missing-prepopulated.csv"
```

Comment:
slight overlap with https://github.com/DFE-Digital/register-trainee-teachers/pull/3770/files#diff-cec521d63c4a5ee83ca40b56f598f2e67b5143db13cc02e566048e850ebf1638R35, wondering if we should extract it